### PR TITLE
Change annoying phrase from content/welcome.html

### PR DESCRIPTION
**Pull request by RIMOPA**
content/welcome.html says `You can also configure each of the addons on the bottom section of each addons.` but that sounds annoying and weird because We say addons two times, this PR modifies it a little bit

### CODE REVIEW

File: content/welcome.html [hunk 1]
```diff
@@ -39,7 +39,7 @@ <h5 class="card-title">Press the gear icon to go to the Settings page</h5>
 			<img class="card-img-top" src="/assets/img/getting-started/step-3.png" alt="Scroll through the addons list and enable the ones that you want">
 			<div class="card-body">
 				<h5 class="card-title">Scroll through the addons list and enable the ones that you want</h5>
-				<p class="card-text">You can also configure each of the addons on the bottom section of each addons.</p>
+				<p class="card-text">You can also configure each of the addons on its bottom section.</p>
```
Comment (WorldLanguages) [inline content/welcome.html:42]: "each addon" maybe?
@RedGuy12

Comment (RedGuy12) [reply content/welcome.html:42]: At the bottom of each adddon's card

Comment (RIMOPA) [reply content/welcome.html:42]: > At the bottom of each adddon's card

Can you send the full sentence? I don't understand

Comment (Norbiros) [reply content/welcome.html:42]: `You can also configure addons at the bottom of each adddon's card`, maybe?

Comment (RIMOPA) [reply content/welcome.html:42]: > `You can also configure addons at the bottom of each adddon's card`, maybe?

The main goal is not repeat "addons", but I think this is the best ay the moment, I'll change it tomorrow.

**Review by Wjplatformer — CHANGES_REQUESTED** [review]
`You can also configure each addon at the bottom of its cards` should be just right

**Comment by RIMOPA** [comment]
> `You can also configure each addon at the bottom of its cards` should be just right

I can't imagine a user thinking a "card" is that. I think this would be confusing.

**Comment by Wjplatformer** [comment]
@RIMOPA , I find it difficult to understand what "card" and "bottom part" even mean. So I was just doing wild guesses on how to change the grammar. Is it fine to send me a screenshot of what you are describing so that I can refine it?

**Comment by RIMOPA** [comment]
> @RIMOPA , I find it difficult to understand what "card" and "bottom part" even mean. So I was just doing wild guesses on how to change the grammar. Is it fine to send me a screenshot of what you are describing so that I can refine it?

There is a screenshot [here](https://scratchaddons.com/welcome/) in the third step. The phrase is also there.

**Comment by RIMOPA** [comment]
> Here is a new version. <br>
> `When you enable an addon by clicking on the bar, the addon bar will expand and you can go to the bottom to configure its settings.`

I personally like it. I will update it as soon as possible or someone can modify it.

**Comment by Secret-chest** [comment]
bar? Also bottom sounds like bottom of the page

**Comment by Wjplatformer** [comment]
Here is how we configure a setting (so as to try to finalize this thing):
Step1
step1addonconfig
Step2
step2addonconfig
Step3
step3addonconfig

**Comment by Hans5958** [comment]
Hey, guys. I may need to merge this to #188 just so we can put things together in a single merge to master. Anyways...

> Here is a new version. `When you enable an addon by clicking on the bar, the addon bar will expand and you can go to the bottom to configure its settings.`

The bottom section can also be expanded manually without turning it on, though.

**Comment by RIMOPA** [comment]
`You can also configure some addons in its settings section, that is opened when you enable an addon or when you expand it.`

**Comment by Secret-chest** [comment]
`You can also configure some addons in their settings section that is opened when you enable an addon or when you expand it.`

**Comment by Wjplatformer** [comment]
`You can also configure an addon in its settings that will open when enabled or expanded`. I prefer it singular. Final?

**Comment by RIMOPA** [comment]
> Hey, guys. I may need to merge this to #188 just so we can put things together in a single merge to master. Anyways...
> 
> > Here is a new version. `When you enable an addon by clicking on the bar, the addon bar will expand and you can go to the bottom to configure its settings.`
> 
> The bottom section can also be expanded manually without turning it on, though.

@Hans5958 Yes! I think the current version will be the definitive one. So wait a little longer and then if no one objects, merge them.

**Review by Wjplatformer — APPROVED** [review]
:)

**Pull request closed without merging**